lib/web/desktop.go

```diff
@@ -453,110 +447,6 @@ func readClientScreenSpec(ws *websocket.Conn) (*tdp.ClientScreenSpec, error) {
 	return tdpConn.ReadClientScreenSpec()
 }
 
-// proxyWebsocketConn does a bidrectional copy between the websocket
-// connection to the browser (ws) and the mTLS connection to Windows
-// Desktop Serivce (wds)
-func proxyWebsocketConn(ws *websocket.Conn, wds net.Conn) error {
-	var closeOnce sync.Once
-	close := func() {
-		ws.Close()
-		wds.Close()
-	}
-
-	errs := make(chan error, 2)
-
-	go func() {
-		defer closeOnce.Do(close)
-
-		// we avoid using io.Copy here, as we want to make sure
-		// each TDP message is sent as a unit so that a single
-		// 'message' event is emitted in the browser
-		// (io.Copy's internal buffer could split one message
-		// into multiple ws.WriteMessage calls)
-		tc := tdp.NewConn(wds)
-
-		// we don't care about the content of the message, we just
-		// need to split the stream into individual messages and
-		// write them to the websocket
-		for {
-			msg, err := tc.ReadMessage()
-			if utils.IsOKNetworkError(err) {
-				errs <- nil
-				return
```

Comment:
This seems to be missing from the new implementation. The new implementation calls `utils.IsOKNetworkError` when reading errors from `errCh`, but this it exhibits different behavior than the old implementation, no?

For example, the old implementation doesn't call `sendTDPAlert` when `utils.IsOKNetworkError` returns true, but the new one does.

Reply:
Oh right, I simplified the original code too much. Thanks for catching this, I brought back the original error handling.